src/entities/BlockNode/types/BlockChildType.ts

```diff
@@ -0,0 +1,4 @@
+export enum BlockChildType {
+  Value = 'v',
+  Text = 't',
```

Comment:
do we need to add type property to all types? Maybe only to ValueNodes would be enough?

Reply:
See https://github.com/editor-js/document-model/pull/39#discussion_r1370877258